Nodes/src/Controller/Admin/NodesController.php

```diff
@@ -344,6 +345,26 @@ public function process() {
 		return $this->BulkProcess->process($this->Nodes, $action, $ids, $options);
 	}
 
+	public function lookup() {
+		$this->Prg->commonProcess();
+
+		/* @var \Cake\Orm\Query $lookup */
+		$lookup = $this->Nodes->find('searchable', $this->Prg->parsedParams());
+		$lookup->contain([
+			'Users', /*'Meta', */'Taxonomies',
+		]);
+		$lookup->select([
+			'id', 'parent_id', 'type', 'user_id', 'title', 'slug',
```

Comment:
one problem with multiple array elements in one line, if one element is later changed, it affects the whole line along with other elements in the diff.